roamance/src/components/common/loading-button.tsx

```
@@ -0,0 +1,47 @@
+  ...props
+}: LoadingButtonProps) {
+  return (
+    <Button disabled={isLoading} {...props}>
+      {isLoading ? (
+        <div className="flex items-center justify-center">
+          <svg
+            className="animate-spin -ml-1 mr-2 h-4 w-4 text-white"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)